### PR TITLE
[OSE-175] GetOperations implementation

### PR DESCRIPTION
**Pull request by andresuribe87**
# Overview
GetOperations implementation. Notably, this doesn't include any filtering logic, which will be added in a subsequent PR. 

# Description

# How Has This Been Tested?
Unit test. 

# Checklist

Before submitting this PR, please make sure:

- [x] I have read the CONTRIBUTING document.
- [x] My code is consistent with the rest of the project 
- [x] I have tagged the relevant reviewers and/or interested parties
- [x] I have updated the READMEs and other documentation of affected packages

## References
See SIP6.

### CODE REVIEW

File: pkg/server/server_operation_test.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestOperationsAPI(t *testing.T) {
+	t.Run("GetOperations", func(t *testing.T) {
+		t.Run("Returns empty when no operations stored", func(t *testing.T) {
```
Comment (decentralgabe) [inline pkg/server/server_operation_test.go:20]: nit: on the inner runners you can use different variables than `t` to not shadow the parent t's

File: pkg/server/server_operation_test.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			value := newRequestValue(t, request)
+			req := httptest.NewRequest(http.MethodGet, "https://ssi-service.com/v1/operations", value)
+			w := httptest.NewRecorder()
+
+			assert.NoError(t, opRouter.GetOperations(newRequestContext(), w, req))
+
+			var resp router.GetOperationsResponse
+			assert.NoError(t, json.NewDecoder(w.Body).Decode(&resp))
+			assert.Empty(t, resp.Operations)
+		})
```
Comment (decentralgabe) [inline pkg/server/server_operation_test.go:35]: newline for readability between these plz

File: pkg/service/operation/service.go [hunk 3]
```diff
@@ -32,6 +32,30 @@ func (s Service) Status() framework.Status {
 	return framework.Status{Status: framework.StatusReady}
 }
 
+func (s Service) GetOperations(request GetOperationsRequest) (*GetOperationsResponse, error) {
+	ops, err := s.storage.GetOperations()
```
Comment (decentralgabe) [inline pkg/service/operation/service.go:36]: could consider a cache on `s Service` in the future

**Review by decentralgabe — APPROVED** [review]
🥳

File: pkg/service/operation/service.go (hunk 3 above)
Comment (andresuribe87) [reply pkg/service/operation/service.go:36]: yes, definitely something we should think about

File: pkg/server/server_operation_test.go (hunk 1 above)
Comment (andresuribe87) [reply pkg/server/server_operation_test.go:20]: I'm of the opinion that shadowing makes sense here, unless there is a strong reason not to do so. If you find yourself having to access the outer `t` from inside a subtest, that's an indication that there is something unusual happening. Curious what your thoughts are?

File: pkg/server/server_operation_test.go (hunk 2 above)
Comment (andresuribe87) [reply pkg/server/server_operation_test.go:35]: Done

File: pkg/server/server_operation_test.go (hunk 1 above)
Comment (decentralgabe) [reply pkg/server/server_operation_test.go:20]: have run into shadowing errs with unexpected state, especially when tests are run in parallel. IMO as much as possible we should make our tests be able to run in parallel, which means eliminating shared state. The test as written is fine - so no problem.

**Pull request merged**